compiler/rustc_session/Cargo.toml

```diff
@@ -30,7 +30,7 @@ tracing.workspace = true
 # FIXME: Remove this pin once this rustix issue is resolved
 # https://github.com/bytecodealliance/rustix/issues/1496
 # tidy-alphabetical-start
```

Comment:
This comment should move to the new location where we specify the version, and the version should remain pinned.